realm/src/androidTest/java/io/realm/RealmTest.java

```diff
@@ -196,21 +196,21 @@ public void testInternalRealmChangedHandlersRemoved() {
         final String REALM_NAME = "test-internalhandlers";
         RealmConfiguration realmConfig = TestHelper.createConfiguration(getContext(), REALM_NAME);
         Realm.deleteRealm(realmConfig);
-        Realm.handlers.clear(); // Make sure that handlers from other unit tests doesn't interfere.
+        Realm.getHandlers().clear(); // Make sure that handlers from other unit tests doesn't interfere.
```

Comment:
Possible, but not sure. This looks like it is used to prevent this test case from failing if other test cases failed first.